test/__snapshots__/github.test.ts.snap

```diff
@@ -398,7 +358,7 @@ jobs:
         uses: actions/download-artifact@v2
         with:
           name: cdk.out
-          path: cdk.out
+          path: /var/folders/5t/qdsk26vd29l6bk_3kg6ydwx00000gs/T/cdk-pipelines-github-WmeGA4/cdk.out
```

Comment:
This diff relates to the issue I've raised above. I don't think we want `path: cdk.out` anymore since that points to the base directory, not the temp directory. This previously wasn't an issue because we didn't actually create + add any files to `cdk.out`.